examples/tf/trpo_cartpole_recurrent.py

```diff
@@ -44,7 +44,7 @@ def trpo_cartpole_recurrent(ctxt, seed, n_epochs, batch_size, plot):
     """
     set_seed(seed)
     with LocalTFRunner(snapshot_config=ctxt) as runner:
-        env = GymEnv('CartPole-v1')
+        env = GymEnv('CartPole-v1', max_episode_length=100)
```

Comment:
Maybe we don't need `max_episode_length=100` here ? According to `GymEnv`, as long as the `env` or `env.spec`has attribute `_max_episode_steps`, it will return maximum time limit equivlent to `max_episode_length`. So in this case, are we trying to override the `max_episode_length` in 'CartPole-v1' env?

Reply:
I checked, 'CartPole-v1' will have `self._ max_episode_length = 500` generated by GymEnv.

Reply:
Yup, I override it because using the default timelimit (500) will increase memory usage and time per optimization step, causing the CI to fail.